.github/workflows/packaging.yml

```diff
@@ -0,0 +1,65 @@
+name: "Build packages"
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Are we building an RPM for every single change to the develop branch?

Reply:
building but not publishing, so it becomes like another test.